Assets/HoloToolkit/Input/Scripts/GestureManipulator.cs

```diff
@@ -47,7 +47,7 @@ private void OnEnable()
 
         private void OnDisable()
         {
-            if (GestureManager.Instance)
+            if (GestureManager.Instance != null)
```

Comment:
Cache GestureManager.Instance.  The .Instance call actually has a small cost that adds up.